stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/QuotedMessageText.kt

```diff
@@ -45,6 +47,13 @@ public fun QuotedMessageText(
 ) {
     val attachment = message.attachments.firstOrNull()
 
+    // TODO: Fix emoji font padding once this is resolved and exposed: https://issuetracker.google.com/issues/171394808
+    val style = when {
+        message.isSingleEmoji() -> ChatTheme.typography.singleEmoji
+        message.isLargeFont() -> ChatTheme.typography.emojiOnly
```

Comment:
Maybe some other name instead of `isLargeFont()`? The message object itself doesn't contain any information about styling. What about `isFewEmoji()`?

Reply:
Agreed - should be either emojiOnly or fewEmoji or something similar to the line above